src/plugins/dam/components/search-assets-screen/index.js

```diff
@@ -98,10 +100,10 @@ function SearchAssetsScreen(props) {
                 const transcodingStatus = node.get('transcoding_status');
 
                 return (
-                  <tr key={`${node.get('_id')}`} className={`status-${node.get('status')}`}>
+                  <tr key={`${node.get('_id')}`} className={`status-${node.get('status')}`} style={{cursor: 'pointer'}}>
                     <td><Input type="checkbox" onChange={() => batch.toggle(node)} checked={batch.has(node)} /></td>
                     <td className="text-center"><AssetIcon id={node.get('_id')} /></td>
-                    <td>
+                    <td onClick={() => navigate(nodeUrl(node, 'view'))}>
```

Comment:
@rsumilang can we make an onClick function and put it on the row (one time).  accept the event and inspect the target to see if it has a data-ignore-row-click.  then put the data-ignore-row-click on the td we don't need a handler for every td?

Reply:
ok check it out